merkletree/tb.go

```diff
@@ -1,6 +1,7 @@
 package merkletree
 
 type TemporaryBinding struct {
+	str   []byte
```

Comment:
Yes. I just realize that the server will return STR proof, VRF proof, proof of absence and TB to the client. So this field should be removed.